docs/basics/getting-started.md

```diff
@@ -1,210 +1,159 @@
-started running a full scan of your torrent directory!
+Or, if you are using Docker, make sure to expose the paths cross-seed needs to
+access:
 
 ```shell
-cross-seed search
-```
-
-### with Docker
-
-#### Set up your container
-
-With Docker, any paths that you would provide as command-line arguments or in
-the config file should stay hardcoded. Instead, you can mount volumes to the
-Docker container.
-
--   `/config` - config dir (this follows a common Docker convention)
--   `/torrents` - your torrent input dir
-    -   usually set to your rTorrent session dir, Deluge state dir, Transmission torrents dir, or your qBittorrent BT_backup dir
--   `/cross-seeds` - the output dir. People sometimes point this at a watch folder. If you use
-
-Create or open your existing `docker-compose.yml` file and add the `cross-seed`
-service:
-
-```yaml
-version: "2.1"
-services:
-    cross-seed:
-        image: ghcr.io/cross-seed/cross-seed
-        container_name: cross-seed
-        user: 1000:1000 # optional but recommended
-        ports:
-            - "2468:2468" # you'll need this for daemon mode only
-        volumes:
-            - /path/to/config/folder:/config
-            - /path/to/torrent_dir:/torrents:ro # your torrent clients .torrent cache, can and should be mounted read-only (e.g. qbit: `BT_Backup` | deluge: `state` | transmission: `transmission/torrents` | rtorrent: session dir from `.rtorrent.rc`)
-            - /path/to/output/folder:/cross-seeds
-            - /path/to/torrent/data:/data # OPTIONAL!!! this is location of your data (used for data-based searches or linking)
-              # will need to mirror your torrent client's path (like Arr's do)
-        command: daemon # this enables the daemon, change to search to specifically run a search ONLY
-        restart: "no" # make sure to not enable it in search mode, otherwise it will keep rerunning constantly
+docker run \
+-v /path/to/config:/config \
+-v /path/to/torrent_dir:/path/to/torrent_dir \
+-v /path/to/output:/path/to/output \
+ghcr.io/cross-seed/cross-seed daemon
 ```
 
-#### Create a config file
-
-When you run the container the first time, it will create a config file at
-`/config/config.js`. Open this file in your favorite editor. Since you've
-already configured your volume mappings, the only required config option is
-[`torznab`](../basics/options.md#torznab).
+After a few moments, you should see `cross-seed` automatically starting to
+search for things in your catalog. It will also automatically scan your
+trackers' RSS feeds for new releases you can cross-seed.
 
-:::tip
-There are many other [config options available](./options.md#all-options) that
-you should thoroughly look through to achieve your desired results.
-:::
+For now, you can leave the terminal open to keep the daemon running. If any
+configuration issues arise, `cross-seed` will provide feedback in the terminal,
+which helps guide you through fixing them.
```

Comment:
```suggestion
For now, you can leave the terminal open to keep the daemon running. If any
configuration issues arise, `cross-seed` will provide feedback in the terminal,
which helps [guide you through fixing them.](./faq-troubleshooting#general-suggestions)
```

Given that a lot of the logging that would actually guide them to fixing this has been relegated to the verbose logs, perhaps linking to the FAQ page would be somewhat more reasonable of a start.

Reply:
I'll just mention it inline.

Reply:
https://github.com/cross-seed/cross-seed.org/pull/58/commits/3b6f8113b5e04d64c78dd54d5d1e4ec451b83e71